packages/core/src/toolkit-provider/ToolkitProvider.tsx

```diff
@@ -191,12 +196,16 @@ export function ToolkitProvider({
   }
 }
 
-export const useTheme = (): Theme[] => {
-  return useContext(ThemeContext);
+export const useTheme = (): ThemeNameType => {
```

Comment:
Is it worth having separate hooks here? They rely on the same context provider so will cause re-render together.

We could return an object from useTheme with { theme, mode }